modules/articles/tests/server/article.server.model.tests.js

```diff
@@ -1,11 +1,15 @@
 'use strict';
 
+var path = require('path');
+var User = require(path.resolve('./modules/users/server/models/user.server.model'));
+var Article = require(path.resolve('./modules/articles/server/models/article.server.model'));
+
 /**
  * Module dependencies.
  */
 var should = require('should'),
-	mongoose = require('mongoose'),
-	User = mongoose.model('User'),
+	mongoose = require('mongoose');
```

Comment:
Without this change the build fails and gives the error

  1) Article CRUD tests "before all" hook:
     Error: timeout of 2000ms exceeded. Ensure the done() callback is being called in this test.
      at Context.<anonymous> (modules/articles/tests/server/article.server.routes.tests.js:25:3)
Warning: Task "mochaTest:src" failed. Used --force, continuing.

Alos this fixes the issue #312 In 0.4.0 and master getting "Cannot find module when running mocha test" on grunt or gulp.

Please let me know if we should keep the change.